OpenRA.Game/Sync.cs

```diff
@@ -62,15 +67,13 @@ static void EmitSyncOpcodes(Type type, ILGenerator il)
 				il.Emit(OpCodes.Ldc_I4, 0x555);
 				il.MarkLabel(l);
 			}
-			else if (type.HasAttribute<SyncAttribute>())
```

Comment:
The trait lookup isn't gone, just cached at https://github.com/OpenRA/OpenRA/pull/10657/files#diff-25e3f19d154f93bcb747e9f12c252028R126.